harness-action_subscriber.gemspec

```diff
@@ -20,7 +20,9 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.10"
   spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "rspec", "~> 3.0"
 
+  spec.add_runtime_dependency "activesupport", ">= 3.2"
```

Comment:
This makes the deps more explicit. This gem will fail to load without active support notifications.